x-pack/package.json

```diff
@@ -54,6 +54,7 @@
     "@types/react-dom": "^16.0.5",
     "@types/react-redux": "^6.0.6",
     "@types/react-router-dom": "^4.3.1",
+    "@types/recompose": "^0.26.0",
```

Comment:
Not with this PR, but sometime soon we should probably figure out the cadence schedule of package updates since `recompose` is at `0.30.0` and they just updated `hapi.js` and we're sitting at `0.26.0`

Newer recompose looks like `0.30.0`:
https://github.com/acdlite/recompose/releases

Wrote an issue for the [backlog](https://github.com/elastic/ingest-dev/projects/9)